src/dynode/__init__.py

```diff
@@ -41,15 +41,15 @@
 
 # Defines all the different modules able to be imported from src
 __all__ = [
-    AbstractParameters,
-    AbstractInitializer,
-    CovidSeroInitializer,
-    MechanisticInferer,
-    MechanisticRunner,
-    SolutionInterpreter,
-    StaticValueParameters,
-    utils,
-    Config,
-    vis_utils,
-    AbstractDynodeRunner,
+    "AbstractParameters",
```

Comment:
`SEIC_Compartments` is missing from `__all__` the list and `SEIC_Timeseries` isn't used anywhere in the code (should be deleted if removed in from code and not used outside of this repo). I am not aware of a reason these `SEIC_Compartments` shouldn't be exposed for import via module.